shell/app-shell/elements/arc-footer.js

```diff
@@ -131,15 +127,22 @@ class ArcFooter extends Xen.Base {
       this._commitSearch(e.target.value || '*');
     }
   }
+  // 4. using speech input
+  _onSpeechInput(e, transcript) {
+    this._setState({search: transcript});
+  }
+  _onSpeechEnd(e, transcript) {
+    this._commitSearch(transcript);
+  }
   _onSearchCommit(e) {
     this._commitSearch(e.target.value);
   }
   _commitSearch(search) {
     search = search || '';
-    if (this._state.search !== search) {
+    //if (this._state.search !== search) {
```

Comment:
Remove?

Reply:
Not sure if it's harmful yet, so thought I would leave it there. Will add comment.